### ISSUE TITLE
black cutscenes

### ISSUE THREAD

**Issue by atesin** [UnknownException/SubTitans#6]
hi...  i dont know if this unofficial patch has some todo with this ... but as this is about of fixes maybe yes

game seem to work fine, but cutscenes are black

- videos with sound but no image
- screen flickers as it were changed resolution, but still no video
- videos play ok directly from ./video/ folder with external players/codecs
- i had tried many versions (currently i have gog and fx interactive (spanish)) with many downloaded dlls
- i like this game

**Comment by UnknownException** [comment]
Could you verify if this issue persists with the latest release?
[2021-12-06](https://github.com/UnknownException/SubTitans/releases/download/2021-12-06_16-43-45/SubTitans-2021-12-06_16-43-45.zip)

**Comment by atesin** [comment]
same thing

- clarify: with each cutscene screen flickers many times as it were TRYING TO change resolution, but still BLACK video with sound at the end

**Comment by atesin** [comment]
wrong recipient... just delete

2021-12-09 19:19 GMT-03:00, Atesin Dj ***@***.***>:

**Comment by atesin** [comment]
could be someting related with codecs? ...  (some ancient games versions like system shock 2 v0.000...1 required an old intel indeo codec installed).... or with bink video ???  (.bik files? ... bink codecs?? .. idk) ....  or directdraw, directplay, directshow...  who knows

**Comment by rebb** [comment]
Maybe this helps : https://www.pcgamingwiki.com/wiki/Submarine_Titans#Broken_video_playback.2C_showing_as_black_screen

**Comment by UnknownException** [comment]
Are you willing to test the cutscenes again with the latest release?  I’d like to know if they render with the OpenGL or Software renderer (subtitans.ini).

**Comment by atesin** [comment]
yes, i tested 2 versions of the game.... without and with this patch

- gog version:
  - before patch: black cutscenes
  - after cutscene: black cutscenes
- fx interactive iso version (spanish distributor repack)
  - before patch: i forgot to test it, but if i delete patch files and try to run the game it complains about `d3drm.dll` not found
  - after patch: all videos play flawlessly ingame (see notes below)

### notes

- in patched fx version, some videos play centered with a thick black border, some others play stretched
- i tested in my desktop pc, with a geforce gtx... i didn't tested in my laptop yet, could still fail because nvidia optimus (i hate it)

**Comment by UnknownException** [comment]
> yes, i tested 2 versions of the game.... without and with this patch
> 
>     * gog version:
>       
>       * before patch: black cutscenes
>       * after cutscene: black cutscenes
> ...

Have you tried using the OpenGL and/or Software renderer with the GOG version? You can configure it in subtitans.ini with the latest release. [Release (2022-05-28)](https://github.com/UnknownException/SubTitans/releases/download/2022-05-28_09-17-33/SubTitans-2022-05-28_09-17-33.zip)
